ozaria/site/views/play/level/LevelPlaybackView.coffee

```diff
@@ -0,0 +1,420 @@
+require('app/styles/play/level/level-playback-view.sass')
+CocoView = require 'views/core/CocoView'
+template = require 'templates/play/level/level-playback-view'
```

Comment:
For these moved files, is it in scope of the PR to connect the template to the moved file. I believe this will point to the legacy template folder instead of the new ozaria template folder?

Reply:
Great point! 

A lot of this will be ripped out, replaced or not used. For now, having it build and render properly is enough. As we progress through each area of the level UX, we can address the template references.